### PR TITLE
chalk update

### PR DESCRIPTION
**Pull request by ghost**
Updated example for chalk

### CODE REVIEW

File: src/app_font_viewer.c [hunk 1]
```diff
@@ -32,23 +33,33 @@ static void up_single_click_handler(ClickRecognizerRef recognizer, void *context
...
   layer_add_child(window_layer, text_layer_get_layer(s_text_layer));
 
-  s_char_text_layer = text_layer_create((GRect) { { PADDING, bounds.size.h - 60 }, { bounds.size.w - 2*PADDING, 40 } });
+  s_char_text_layer = text_layer_create((GRect) { { X_PADDING, bounds.size.h - 60 }, 
+                                                  { bounds.size.w - 2 * X_PADDING, 40 } });
   text_layer_set_text(s_char_text_layer, s_text_buffer);
   text_layer_set_font(s_char_text_layer, fonts_get_system_font(FONT_KEY_BITHAM_30_BLACK));
   layer_add_child(window_layer, text_layer_get_layer(s_char_text_layer));
+
+#ifdef PBL_ROUND
```
Comment (C-D-Lewis) [inline src/app_font_viewer.c]: Replace these with `PBL_IF_ROUND_ELSE()` inline.

**Comment by C-D-Lewis** [comment]
Thanks! Aside from single comment, :+1: Update the spreadsheet when you're done with each of these.

**Comment by ghost** [comment]
@C-D-Lewis Code review changes made. Merge to master?

**Comment by C-D-Lewis** [comment]
:ship:

**Pull request merged**